### PR TITLE
Add skipped test for https://github.com/dotnet/roslyn/issues/320

### PR DESCRIPTION
**Pull request by mavasani**
Add skipped test for https://github.com/dotnet/roslyn/issues/320: Possible bug in Batch Fixer merge algorithm.

### CODE REVIEW

**Comment by srivatsn** [comment]
:+1:

File: src/EditorFeatures/CSharpTest/Diagnostics/FixAllProvider/BatchFixerTests.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                    if (symbol != null && symbol.Kind == SymbolKind.Field)
+                    {
+                        var diagnostic = Diagnostic.Create(Descriptor, node.GetLocation());
+                        context.ReportDiagnostic(diagnostic);
+                    }
+                }
+            }
+        }
+
+        class QualifyWithThisFixer : CodeFixProvider
```
Comment (jaredpar) [inline src/EditorFeatures/CSharpTest/Diagnostics/FixAllProvider/BatchFixerTests.cs]: private

File: src/EditorFeatures/CSharpTest/Diagnostics/FixAllProvider/BatchFixerTests.cs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                        node.WithoutLeadingTrivia())
+                        .WithLeadingTrivia(leadingTrivia);
+
+                    var newRoot = root.ReplaceNode(node, newNode);
+                    var newDocument = context.Document.WithSyntaxRoot(newRoot);
+
+#pragma warning disable RS0005 // Do not use generic CodeAction.Create to create CodeAction
+                    var fix = CodeAction.Create("QualifyWithThisFix", _ => Task.FromResult(newDocument));
+#pragma warning restore RS0005 // Do not use generic CodeAction.Create to create CodeAction
+
```
Comment (jaredpar) [inline src/EditorFeatures/CSharpTest/Diagnostics/FixAllProvider/BatchFixerTests.cs]: Can we add a justification on why we are doing this?

**Comment by jaredpar** [comment]
LGTM

File: src/EditorFeatures/CSharpTest/Diagnostics/FixAllProvider/BatchFixerTests.cs (hunk 2 above)
Comment (mavasani) [inline src/EditorFeatures/CSharpTest/Diagnostics/FixAllProvider/BatchFixerTests.cs]: Sure.

**Pull request merged**